### DISCUSSION TITLE
Support for JSON type in nexus schema plugin prisma

### DISCUSSION THREAD

**Discussion by felinto-dev** [prisma/prisma#2649] [Q&A] [answered]
My schema:
```
model Invoice {
  id                Int                  @default(autoincrement()) @id
  status            InvoiceStatus
  transactions      PaymentTransaction[]
  purchaseDate      DateTime
  details           Json
  products          Product[]
  supportExpiration DateTime
  customerId        Int
  customer          Customer             @relation(fields: [customerId], references: [id])
  sellerId          Int
  seller            Seller               @relation(fields: [sellerId], references: [id])
}
```

Invoice.ts Nexus Prisma Type

```
import { objectType } from '@nexus/schema'

export const Invoice = objectType({
  name: 'Invoice',
  definition(t) {
    t.model.id()
    t.model.details()
  },
})
```

https://prnt.sc/sshpdt
https://prnt.sc/sshq1h

This error should not occur because "Json" is not exactly a type that makes sense to be added to the Nexus Prisma.

Is there any solution for this ?

**Comment by janpio** [comment, marked as answer]
You posted the same text over at https://github.com/graphql-nexus/nexus-schema-plugin-prisma/issues/680

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
